src/main/java/core/basesyntax/Application.java

```diff
@@ -4,11 +4,13 @@
     public static void main(String[] args) {
         Lottery lottery = new Lottery();
         Ball[] lotteryResult = new Ball[3];
-        for (int i = 0; i < 3; i++) {
+        for (int i = 0; i < lotteryResult.length; i++) {
             lotteryResult[i] = lottery.getRandomBall();
-            System.out.println("It`s " + lotteryResult[i].getColor() + " ball number "
-                    + lotteryResult[i].getNumber() + "!");
+            System.out.println(lotteryResult[i].toString());
```

Comment:
```suggestion
            System.out.println(lotteryResult[i]);
```
toString() will be called automatically

Reply:
lottery unlucky(